src/main/java/com/auth0/json/mgmt/EmailVerificationIdentity.java

```diff
@@ -0,0 +1,52 @@
+package com.auth0.json.mgmt;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+
+/**
+ * Represents the identity object that can be sent on requests to create an email verification ticket or job.
+ * Related to {@link com.auth0.client.mgmt.JobsEntity} and {@link com.auth0.client.mgmt.TicketsEntity}.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class EmailVerificationIdentity implements Serializable {
+
+    @JsonProperty("provider")
+    private String provider;
+    @JsonProperty("user_id")
+    private String userId;
+
+    /**
+     * Create a new instance.
+     *
+     * @param provider The identity provider name (e.g. {@code "google-oauth2"}).
+     * @param userId The user ID of the identity to be verified.
+     */
+    public EmailVerificationIdentity(String provider, String userId) {
+        this.provider = provider;
+        this.userId = userId;
+    }
+
+    /**
+     * Get the identity provider name.
+     *
+     * @return The identity provider name.
+     */
+    @JsonProperty("provider")
+    public String getProvider() {
+        return this.provider;
+    }
+
+    /**
+     * Get the user ID.
+     *
+     * @return the User ID.
+     */
+    @JsonProperty("user_id")
+    public String getUserId() {
```

Comment:
The main method you added in the API client already receives a user id. What is this one for? Should it be renamed?

Reply:
The `identity` object is defined as:
```
"identity": {
    "provider": "google-oauth2",
    "user_id": "117029267717860381713"
}
```

The `user_id` of the identity is required. For example, in the case of verifying a secondary email address, the `user_id` of the `identity` will be different than the top-level `user_id` field. For example, from the [send email verification job API docs](https://auth0.com/docs/api/management/v2#!/Jobs/post_verification_email):

```
{
  "user_id": "google-oauth2|1234",
  "client_id": "",
  "identity": {
    "user_id": "5457edea1b8f22891a000004",
    "provider": "google-oauth2"
  }
}
```